src/dashboard/components/Dashboard/Catalog/Products/Editor/ProductEditorWrapper.tsx

```diff
@@ -393,6 +396,45 @@ const ProductEditorWrapper = ({
           </Alert>
         </Snackbar>
       ) : null}
+      <Grid container spacing={2}>
+        <Grid item md={8} xs={12}>
+          <EditorCard>
+            <Box>
+              <Button
+                variant="contained"
+                onClick={() => {
+                  if (!productData) {
+                    return;
+                  }
+                  setPreventNavigation(false);
```

Comment:
Shouldnt there be `await setPreventNavigation(false);` ? I remmember it caused some troubles in the past.

Reply:
Yes, good point! It could cause an issue, since `setPreventNavigation` is asynchrnous